homework/index.js

```diff
@@ -44,6 +74,7 @@
         return;
       }
       const ul = createAndAppend('ul', root);
+      repos.sort(sorting);
       repos.forEach(repo => renderRepoDetails(repo, ul));
```

Comment:
You can also do the sort in-line using an arrow function and then chain the `.forEach()`:

```js
repos
  .sort((a, b) => a.name.localeCompare(b.name))
  .forEach(repo =>  renderRepoDetails(repo, ul));
```

Reply:
done :+1: